.circleci/config.yml

```diff
@@ -13,8 +13,39 @@ jobs:
       - run: cd svc-add && dep ensure
       - run: cd svc-string && dep ensure
       - run: go test $(go list ./... | grep -v /vendor/)
+  build-and-deploy:
+    <<: *defaults
+    environment:
+      CLOUDSDK_CORE_DISABLE_PROMPTS: 1
+    steps:
+      - checkout
+      - run:
+          name: Install gcloud
+          command: |
+            curl https://sdk.cloud.google.com | bash
+            ${HOME}/google-cloud-sdk/bin/gcloud components install kubectl
```

Comment:
Is there an image with `kubectl` installed?

Reply:
Right now to build and deploy we need `docker` and `gloud` which we cannot find image that has both. What we've tried is `google/cloud-sdk` image but sadly it doesn't have docker.

Reply:
Okie